<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modWidget</class>
 <widget class="QWidget" name="modWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* modWidget */
#modWidget {
  background-color: rgba(255, 255, 255, 0.1);
  border-radius: 10px;
}

/* modWidget Label */
#modWidget QLabel {
  color: #DCE4EE;
  background-color: rgba(255, 255, 255, 0.0);
}

/* modWidget PushButton */
#modWidget QPushButton {
  background-color: #9e2c24;
  border-radius: 5px;
}
#modWidget QPushButton:pressed {
  background-color: #6b1d18;
}</string>
  </property>
  <widget class="QLabel" name="modName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Mod name</string>
   </property>
   <property name="text">
    <string>Mod name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actionButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../resources/trash.png</normaloff>../../../../resources/trash.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="modIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>../../../../resources/no-icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="modDetails">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>File: this-is-a-test.jar</string>
   </property>
   <property name="text">
    <string>File: this-is-a-test.jar</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
